packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -163,13 +163,15 @@ export abstract class PersistentStream<
 
   constructor(
     private queue: AsyncQueue,
+    backoffTimerId: TimerId,
```

Comment:
This name makes it sound like you want to use different IDs for operations that get backed off. I don't think we should do that, and instead use the same ID regardless of how it gets scheduled. 
Do you mind changing this to `connectionTimerId` (or something similar) to indicate the operation type rather than the fact that it is can be run with backoff?